tests/generation/test_utils.py

```diff
@@ -2533,11 +2533,11 @@ def test_transition_scores_greedy_search_normalized(self):
         transition_scores = model.compute_transition_scores(outputs.sequences, outputs.scores, normalize_logits=True)
         expected_scores = np.array(
             [
-                [-6.5532393, -6.5158753, -6.451863, -6.4527144, -6.459402],
-                [-6.5685124, -6.4277077, -6.282607, -6.399295, -6.340927],
+                [-2.538938, -2.2694316, -2.1580915, -1.572299, -2.6719835],
+                [-1.8826028, -2.2461371, -1.7556462, -2.9644494, -1.7996008],
             ]
         )
-        self.assertTrue(np.allclose(transition_scores.cpu().numpy(), expected_scores))
+        self.assertTrue(np.allclose(transition_scores.cpu().numpy(), expected_scores, atol=1e-4))
```

Comment:
`atol` needed for CPU<>GPU output mismatches (the hardcoded values are correct on GPU)